versioned_docs/version-2/helm/helm_get.md

```diff
@@ -55,10 +52,10 @@ helm get [flags] RELEASE_NAME
 
 ### SEE ALSO
 
-* [helm](./helm.md)	 - The Helm package manager for Kubernetes.
-* [helm get hooks](./helm_get_hooks.md)	 - Download all hooks for a named release
-* [helm get manifest](./helm_get_manifest.md)	 - Download the manifest for a named release
-* [helm get notes](./helm_get_notes.md)	 - Displays the notes of the named release
-* [helm get values](./helm_get_values.md)	 - Download the values file for a named release
+* [helm](helm.md)	 - The Helm package manager for Kubernetes.
+* [helm get hooks](helm_get_hooks.md)	 - Download all hooks for a named release
+* [helm get manifest](helm_get_manifest.md)	 - Download the manifest for a named release
+* [helm get notes](helm_get_notes.md)	 - Displays the notes of the named release
```

Comment:
helm_get_notes.md is a broken link

In the live v2 site, Helm get notes is here https://v2.helm.sh/docs/helm/#helm-get-notes 

But it doesn’t look like a helm_get_notes.md file exists in our repo

SOLUTION: 
Option 1) move the existing helm_get_notes.md file  to the new version-2 folder. 
Option 2) create a new helm_get-notes.md file if one doesn’t exist. 
Option 3) Could also remove this link if helm_get_notes shouldn’t exist